spec.html

```diff
@@ -890,6 +890,30 @@ <h1>AsyncContext.Snapshot.prototype</h1>
       </emu-clause>
     </emu-clause>
 
+    <emu-clause id="sec-function-properties-of-the-asynccontext-snapshot-object">
+      <h1>Function Properties of the AsyncContext.Snapshot Object</h1>
+
+      <emu-clause id="sec-asynccontext-wrap">
+        <h1>AsyncContext.Snapshot.wrap ( _fn_ )</h1>
+        <p>This function returns a new function which restores the current value of all AsyncContext.Variable values when being invoked.</p>
+
+        <emu-alg>
+          1. If IsCallable(_fn_) is false, throw a TypeError exception.
+          1. Let _snapshot_ be AsyncContextSnapshot().
+          1. Let _closure_ be a new Abstract Closure with parameters (..._args_) that captures _snapshot_ and performs the following steps when called:
```

Comment:
```suggestion
          1. Let _closure_ be a new Abstract Closure with parameters (..._args_) that captures _snapshot_ and _fn_ and performs the following steps when called:
```

Btw, it seems like ecmarkup's linter doesn't like rest parameters in abstract closures.

Reply:
Done.